Packages/Sites/Sfi.Kateheo/Configuration/NodeTypes.yaml

```diff
@@ -82,6 +82,60 @@ blah: &alohaRich
       'Sfi.Listable:ListableMixin': true
       'Sfi.Listable:ListableCategoryMixin': true
 
+'Sfi.Kateheo:Author':
+  superTypes:
+    - 'TYPO3.Neos:Document'
+  ui:
+    label: 'Автор'
+  childNodes:
+    main:
+      type: 'TYPO3.Neos:ContentCollection'
+  properties:
+    title:
+      ui:
+        inlineEditable: true
+        aloha:
+          <<: *alohaPlain
+          placeholder: 'Имя автора'
+    originalIdentifier:
+      type: string
+
+'Sfi.Kateheo:TagsPage':
+  superTypes:
+    - 'TYPO3.Neos:Document'
+    - 'Sfi.Listable:ListableCategoryMixin'
+  ui:
+    label: 'Страница тэгов'
+  childNodes:
+    main:
+      type: 'TYPO3.Neos:ContentCollection'
+  properties:
+    originalIdentifier:
```

Comment:
The sole purpose of this property was to map old news article ids from old TYPO3 database, to create auto redirects. So it only makes sense to have it on existing nodetypes, that were migrated from old db via the script.